docker/compose/indexer-grpc/cache-worker-config.yaml

```diff
@@ -5,4 +5,4 @@ server_config:
   file_store_config:
     file_store_type: LocalFileStore
     local_file_store_path: /opt/aptos/file-store
-  redis_main_instance_address: 172.16.1.12:6379
+  redis_main_instance_address: 172.16.1.12:6379 # use the primary
```

Comment:
should these be hardcoded? Should we be reading these from some place?

Reply:
they're hard coded from the base docker-compose, where we're using static IPs